num-utils.asd

```diff
@@ -43,15 +43,46 @@
    (:file "test-utilities")
    (:file "pkgdcl")))
 
+(defsystem "num-utils/static-dispatch"
+  :description "Numerical utilities for Common Lisp enhanced by STATIC-DISPATCH."
+  :depends-on (#:anaphora
+               #:alexandria
+               #:alexandria+
+               #:array-operations
+               #:select
+               #:let-plus
+               #:static-dispatch)
+  :pathname "static-dispatch/"
+  :serial t
+  :components
+  ((:file "packages")
+   (:file "utilities")
+   (:file "num=")
+   (:file "arithmetic")
+;;   (:file "arithmetic-type") ; now in src/old/ Looks like it was a WIP
+   (:file "elementwise")
+   (:file "extended-real")
+   (:file "interval")
+   (:file "print-matrix")
+   (:file "matrix")
+   (:file "matrix-shorthand")
+   (:file "chebyshev")
+   (:file "polynomial")
+   (:file "rootfinding")
+   (:file "quadrature")
+   (:file "log-exp")
+   (:file "test-utilities")
+   (:file "pkgdcl")))
+
 (defsystem "num-utils/tests"
   :version "1.0.0"
   :description "Unit tests for NUM-UTILS."
   :author "Steven Nunez <steve@symbolics.tech>"
   :license "Same as NUM-UTILS -- this is part of the NUM-UTILS library."
   #+asdf-unicode :encoding #+asdf-unicode :utf-8
-  :depends-on (#:num-utils
+  :depends-on (#:num-utils/static-dispatch
```

Comment:
Is this temporary? It looks like this replaces all the num-util tests with the ones from static-dispatch.  Ideally we could keep two test systems until static-dispatch is ready to be folded into the main system.

Reply:
Oops, yes, my bad! I will fix it.